modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -127,6 +129,7 @@ public ObjectDefinitionDeployerImpl(
 		_modelSearchRegistrarHelper = modelSearchRegistrarHelper;
 		_objectDefinitionLocalService = objectDefinitionLocalService;
 		_objectEntryLocalService = objectEntryLocalService;
+		_objectEntryService = objectEntryService;
 		_objectEntryManagerTracker = objectEntryManagerTracker;
```

Comment:
@marcelabc I think the `objectEntryService` should come after the `objectEntryManagerTracker`